wolfcrypt/test/test.c

```diff
@@ -30873,7 +30873,8 @@ static wc_test_ret_t ecc_test_curve_size(WC_RNG* rng, int keySize, int testVerif
     /* WC_BIGINT doesn't have 0 len well on hardware */
     /* Cryptocell has issues with all 0 digest */
 #if defined(ECC_SHAMIR) && !defined(WOLFSSL_ASYNC_CRYPT) && \
-    !defined(WOLFSSL_CRYPTOCELL)
+    !defined(WOLFSSL_CRYPTOCELL) && \
```

Comment:
I wonder if ECC_SHAMIR is actually still required. Also it would be great to have settings.h set WC_TEST_NO_ECC_SIGN_VERIFY_ZERO_DIGEST if WOLFSSL_ASYNC_CRYPT or WOLFSSL_CRYPTOCELL

Reply:
fixed